src/graphql/helpers.ts

```diff
@@ -124,27 +118,67 @@ export function formatSpace({
   return space;
 }
 
-export function buildWhereQuery(fields, alias, where) {
+export function formatAddresses(
+  addresses: string[],
+  types: ('evmAddress' | 'starknetAddress')[] = [
+    'evmAddress',
+    'starknetAddress'
+  ]
+) {
+  return addresses
+    .map(address => {
+      const results: string[] = [];
+
+      if (
+        types.includes('evmAddress') &&
+        snapshot.utils.isEvmAddress(address)
+      ) {
+        results.push(snapshot.utils.getFormattedAddress(address, 'evm'));
+      } else if (
+        types.includes('starknetAddress') &&
+        snapshot.utils.isStarknetAddress(address)
+      ) {
+        results.push(snapshot.utils.getFormattedAddress(address, 'starknet'));
+      }
+
+      return results;
+    })
+    .flat()
+    .filter(Boolean);
```

Comment:
I think it will be same with out these 🤔 
```suggestion
    });
```